File: Fragments/app/src/main/java/org/guildsa/fragments/TextFragment.java
package org.guildsa.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.fragment.app.Fragment;

public class TextFragment extends Fragment {

    private TextView textView;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        View view = inflater.inflate(R.layout.text_fragment, container, false);

        textView = (TextView) view.findViewById(R.id.textView1);

        return view;
    }

    public void changeTextProperties(int fontsize, String text) {

        textView.setTextSize(fontsize);
        textView.setText(text);
    }
}

File: Fragments/app/src/main/java/org/guildsa/fragments/TextFragment.kt
package org.guildsa.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment

class TextFragment : Fragment() {
    private var textView: TextView? = null
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.text_fragment, container, false)
        textView = view.findViewById<View?>(R.id.textView1) as TextView?
        return view
    }

    fun changeTextProperties(fontsize: Int, text: String?) {
        textView.setTextSize(fontsize.toFloat())
        textView.setText(text)
    }
}